lighthouse-core/audits/dobetterweb/dom-size.js

```diff
@@ -31,7 +31,7 @@ const UIStrings = {
     `depth < ${MAX_DOM_TREE_DEPTH} elements and fewer than ${MAX_DOM_TREE_WIDTH} ` +
     'children/parent element. A large DOM can increase memory usage, cause longer ' +
     '[style calculations](https://developers.google.com/web/fundamentals/performance/rendering/reduce-the-scope-and-complexity-of-style-calculations), ' +
```

Comment:
Just curious if `style calculations` and `layout reflows` will be replaced sometime?

Reply:
Probably! 😄 

@addyosmani can probably give a better sense of the priority of those docs relative to other stuff in the perf section.